### PR TITLE
[KYUUBI #479][TEST][FOLLOWUP] Bind host to 127.0.0.1 in initialize

### PR DESCRIPTION
**Pull request by turboFei**
turboFei [Closes #489](https://github.com/yaooqinn/kyuubi/pull/489)    [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_
Bind host to 127.0.0.1 in initialize phase, so that even some  suite override the beforeAll method and invoke super.beforeAll after start service.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by turboFei** [comment]
```
SparkOperationSuite:
*** RUN ABORTED ***
  java.net.BindException: Cannot assign requested address: Service 'sparkDriver' failed after 16 retries (on a random free port)! Consider explicitly setting the appropriate binding address for the service 'sparkDriver' (for example spark.driver.bindAddress for SparkDriver) to the correct binding address.
  at sun.nio.ch.Net.bind0(Native Method)
  at sun.nio.ch.Net.bind(Net.java:461)
  at sun.nio.ch.Net.bind(Net.java:453)
  at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:222)
  at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134)
  at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:550)
  at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334)
  at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
  at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
  at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973)
```

**Review by yaooqinn — APPROVED** [review]

File: kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala [hunk 1]
```diff
@@ -35,9 +35,9 @@ trait KyuubiFunSuite extends FunSuite
   with ThreadAudit
   with Logging {
   // scalastyle:on
```
Comment (ulysses-you) [inline kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala:37]: not relate about this pr, why here forbid scalastyle check ？

**Comment by pan3793** [comment]
There are some issues on GitHub Action this days, and since travis-CI passed, I will merge it. Thx.

File: kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala (hunk 1 above)
Comment (pan3793) [reply kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala:37]: Let me try to remove it.

Comment (pan3793) [reply kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiFunSuite.scala:37]: The forbid is necessary because of the rule:
```
  <check customId="funsuite" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
    <parameters><parameter name="regex">^FunSuite[A-Za-z]*$</parameter></parameters>
    <customMessage>Tests must extend org.apache.kyuubi.KyuubiFunSuite instead.</customMessage>
  </check>
```
